src/canvas/quadFeature.js

```diff
@@ -174,8 +174,8 @@ var canvas_quadFeature = function (arg) {
         if (!quad.crop) {
           context2d.drawImage(src, 0, 0);
         } else {
-          var cropx = Math.min(src.w, quad.crop.x),
-              cropy = Math.min(src.h, quad.crop.y);
+          var cropx = Math.min(w, quad.crop.x),
```

Comment:
what is the difference between src.w and w? do they point to different values?

Reply:
`w` is a local variable that was taken from `src.width` or `src.videoWidth`.  `src.w` was incorrect.

Reply:
got it, thans.